docs/designs/repository-urls.md

```diff
@@ -0,0 +1,102 @@
+# Repository URLs
+
+Explain the 3 types of repository urls used in DocFX and their design purposes and use scenarios.   
```

Comment:
When I look at the scenarios again, I am thinking whether it'd be clearer to have below 3:
`Repository Url`
`Publish Repository Url`
`Pull Request Repository Url` (which is consistent with https://review.docs.microsoft.com/en-us/engineering/projects/feature-utilization/query-events-and-metrics?branch=live#buildinfo-properties)

What do you think @928PJY @freewheel70 ?

Reply:
synced offline